### DISCUSSION TITLE
Copilot autocomplete conflict with Jetbrain IDEs live templates

### DISCUSSION THREAD

**Discussion by Brakkar** [community/community#62891] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Hi,
Jetbrain IDEs live template autocompletion never work since github copilot was installed: the live template autocomplete is always superseded by Copilot autocomplete.

I suggest copilot autocomplete check if live template is triggered by what is typed, and if yes, gives priority to live template.

**Comment by Akash1134** [comment, marked as answer]
Hi @Brakkar ,

Have you can tried changing the keyboard shortcut or code snippet in any one of them? For example, you could change the keyboard shortcut for GitHub Copilot's autocomplete feature to something that doesn't conflict with the live templates feature in your JetBrains IDE.

This has been reported earlier with other IDE's so I will check with our team for more details.

Thanks!

**Reply by Brakkar** [reply]
Hi @Akash1134 ,

indeed, changing the shortcut of the live template did the trick.
I think it can be considered a fix for this issue. Thank you.

**Reply by dieend** [reply]
What keyboard shortcut you were replacing to make it work? I updated copilot to apply with ctrl + alt + enter, but copilot doesn't do anything when live-template is active/filling in the variables.

--
edit: I think I misunderstood the main issue. My issue is I can't get copilot to fill in the blanks in the middle of using live templates.

**Reply by rwese** [reply]
I can also say, it does not work properly, I was using Ctrl+G to accept copilot suggestions, but simply trying to accept a suggestions during filling an `if ([live-template-field Ctrl+G]) {...` will not work.

**Comment by SumanS7865** [comment]
Response #2
